web/packages/plugins/apps/analysis-platform/src/modules/Asset/components/AssetInfo.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react'
+
+import { AuthContext, hasExpertRole } from '@dmt/common'
+import {
+  CreateAnalysisButtonProps,
+  TAsset,
+  TAssetInfoCardProps,
+} from '../../../Types'
+import { DocumentInfoCard } from '../../../components/DocumentInfoCard'
+import { CreateAnalysisButton } from '../../Analysis'
+import { OnRight } from '../../../components/Design/Styled'
+
+const assetInfoCardFields = (asset: TAsset) => {
+  return {
+    Contact: asset.contact || '',
+    Location: asset.location?.label || asset.location?.name || '',
+  }
+}
```

Comment:
Don't think this should be a separate function. But rather be inline since it's used only once. Better readability